clicache/acceptance-test/SNITests.cs

```diff
@@ -90,7 +91,18 @@ private int ParseProxyPort(string proxyString)
             return Int32.Parse(portNumberString);
         }
 
-        [Fact]
+
+		//private Task PutAsync(IRegion<string, string> region)
```

Comment:
Alas, poor commented code! I knew it, @mmartell : a line of async puts, of most excellent fancy: it hath borne me on its back a thousand times; and now, how abhorred in my imagination it is!

Reply:
Lifted me for at least the remainder of this day you have. 👍